packages/file_testing/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ environment:
   sdk: '>=3.0.0-325.0.dev <4.0.0'
 
 dependencies:
-  test: ^1.23.1
+  test: ^1.24.0
```

Comment:
This probably can/should stay at ^1.23.1 - flutter pins test_api versions which means flutter users can't update `test` without updating their SDK.